app/backend/src/tests/test_search.py

```diff
@@ -73,3 +77,50 @@ def test_regression_search_in_area(db):
             )
         )
         assert [result.user.user_id for result in res.results] == [user3.id, user4.id]
+
+
+def test_user_filter_complete_profile(db):
+    """
+    Make sure the completed profile flag returns only completed user profile
+    """
+    uploader_user, _ = generate_user()
+    print(uploader_user)
+    with session_scope() as session:
+        key = random_hex(32)
+        filename = random_hex(32) + ".jpg"
+        session.add(
+            Upload(
+                key=key,
+                filename=filename,
+                creator_user_id=uploader_user.id,
+            )
+        )
+        session.commit()
```

Comment:
This commit is not needed when exiting the context manager, it'll commit automatically